app/assets/javascripts/datepicker.js

```diff
@@ -0,0 +1,5 @@
+var onReadyDatePicker = function() {
```

Comment:
Let's get rid of this since we already have a datepicker plugin (see new Medication page)